tests/test_applicants.py

```diff
@@ -18,34 +18,34 @@
 
 
 def test_create_applicant(requests_mock):
-    mock_create = requests_mock.post("https://api.onfido.com/v3/applicants/", json=[])
+    mock_create = requests_mock.post("https://api.onfido.com/v3.1/applicants/", json=[])
```

Comment:
Maybe we should introduce some constant variable for this? Happy for this to be considered as a future improvement and not blocking this change.

Reply:
Agreed on both counts, will make a ticket